pages/system/modals.js

```diff
@@ -0,0 +1,238 @@
+import * as React from "react";
+import * as System from "~/components/system";
+import Group from "~/components/system/Group";
+import * as Constants from "~/common/constants";
+
+import SystemPage from "~/components/system/SystemPage";
+import ViewSourceLink from "~/components/system/ViewSourceLink";
+
+export default class SystemPageNotifications extends React.Component {
+  _handleCreate = (detail) => {
+    let event = new CustomEvent("create-modal", { detail });
+    window.dispatchEvent(event);
+  };
+
+  _handleDelete = () => {
+    let event = new CustomEvent("delete-modal", {});
+    window.dispatchEvent(event);
+  };
+
+  render() {
+    return (
+      <SystemPage
+        title="SDS: Modals"
+        description="..."
+        url="https://fps.onrender.com/system/modals"
+      >
+        <System.H1>
+          Modals <ViewSourceLink file="system/modals.js" />
+        </System.H1>
+        <br />
+        <br />
+        <System.P>
+          The Modal component is used to get a user's focus and require an
+          action.
+        </System.P>
+        <br />
+        <br />
+        <br />
+        <System.H2>Imports</System.H2>
+        <hr />
+        <br />
+        <System.P>Import React and the Modal Component.</System.P>
+        <br />
+        <System.CodeBlock>
+          {`import * as React from 'react';
+import { GlobalModal } from 'slate-react-system';`}
+        </System.CodeBlock>
+        <br />
+        <br />
+        <br />
+        <System.H2>Usage</System.H2>
+        <hr />
+        <br />
+        <System.P>
+          Declare the component at the root level of your document (e.g. in
+          index.js or App.js) so it is accessible throughout and will not get
+          buried in the DOM tree.
+        </System.P>
+        <br />
+        <System.P>
+          Optionally, use <System.CodeText>backgroundStyle</System.CodeText> to
+          style the background and <System.CodeText>style</System.CodeText> to
+          style the modal itself.
+        </System.P>
+        <br />
+        <System.CodeBlock>
+          {`class App extends React.Component {
+  render() {
+    return(
+      <React.Fragment>
+        <GlobalModal style={{ padding: "10px" }}/>
+        {this.props.children}     
+      </React.Fragment>
+    )
+  }
+}`}
+        </System.CodeBlock>
+        <System.GlobalModal style={{ padding: "10px" }} />
+        <br />
+        <br />
+        <br />
+        <System.H2>Modal</System.H2>
+        <hr />
+        <br />
+        <System.ButtonSecondaryFull
+          onClick={() =>
+            this._handleCreate({
+              modal: (
+                <div
+                  style={{
+                    textAlign: "center",
+                    position: "absolute",
```

Comment:
Hmm I find this absolute nesting to be odd, You already have the modal in a fixed container, there shouldn't be an absolute nesting.

Reply:
I was using it moreso to vertically+ horizontally center the text and button inside the modal. When I remove the `position: absolute` and leave it as 
![image](https://user-images.githubusercontent.com/33686587/87483189-f19df680-c5e7-11ea-9fe4-e7a758ea5159.png)
things start to get a bit wonky. I did it this way because `margin: auto` didn't center it vertically. Let me know if you know a better way to center it!
![image](https://user-images.githubusercontent.com/33686587/87483230-14c8a600-c5e8-11ea-8418-f10d0506ad9a.png)